onnxruntime/core/providers/cpu/math/hardmax.h

```diff
@@ -19,9 +19,6 @@ class Hardmax final : public OpKernel {
     if (status.IsOK()) {
       axis_ = gsl::narrow_cast<int>(axis);
     }
```

Comment:
As part of resolving #1814, my PR #1829 includes HardMax and LpNorm. Can you please just take LpNorm as the 6th op in your change, so that I can just close my PR?

Reply:
Fyi- there is no need to register a new kernel for LpNorm. It is just addressing an implementation gap - so you ll just have to take 2-3 lines of code plus test

Reply:
Sure, I will have total 15 ops to cover the Negative Axis case, this is the first batch of change.

I will take LpNorm as well.